### PR TITLE
NPM version fix

### PR DESCRIPTION
**Pull request by shammamah**
Fix `unpkg` URL for NPM package, ensure that only necessary files are uploaded to `unpkg`, and update version accordingly.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,12 +18,13 @@
     "doc-stage": "git add docs",
     "install-local": "yarn copy-lib && python setup.py install",
     "lint": "eslint --fix --ignore-path .eslintignore .",
-    "prepublish": "yarn test && builder run build-dist && yarn copy-lib",
+    "prepublish": "yarn test && builder run build-dist-dev && yarn copy-lib",
```
Comment (Marc-Andre-Rivet) [inline package.json]: Why a dev build in prepublish now?

Comment (shammamah) [reply package.json]: There's some issue with the `build-dist` script, which is what was causing it to break. We can look into it (probably has something to do with the webpack config file used in the script.

Comment (Marc-Andre-Rivet) [reply package.json]: Ok. Please open an issue for fixing the prod build and I'll accept this PR as a temporary fix while figuring out what's happening in our build system.

Comment (shammamah) [reply package.json]: Issue opened: #34

**Pull request merged**